.vib/airflow/runtime-parameters.yaml

```diff
@@ -16,3 +16,6 @@ dags:
       branch: main
       name: toys
       path: dags/toys
+metrics:
+  enabled:
+    true
```

Comment:
enabled: true

Reply:
Could you add a goss `http` test that scrapes the metrics endpoint?

Reply:
done at https://github.com/bitnami/charts/pull/30459/commits/d0edc8287060d33f30716114c1ce77d932587731